docs/software/mqtt/index.mdx

```diff
@@ -96,19 +96,25 @@ The `from` field can thus be used as a stable identifier for a specific node.  N
 
 If the message received contains valid JSON in the payload, the JSON is deserialized and added as a JSON object rather than a string containing the serialized JSON.
 
-**Sent messages** will be checked if the MQTT payload contains a valid JSON-encoded envelope:
+
+#### JSON downlink to instruct a node to send a message
+You can also send a JSON message to the topic `msh/2/json/mqtt/` to instruct a gateway node to send a message to the mesh. Note that the channel **must** be called "mqtt". The JSON message should contain the following fields:
 
 ```json
 {
-   "sender": "SENDER",
+   "from": <node number>,
+   "type": "type",
    "payload": {
        "key":"value"
        ...
     }
 }
 ```
 
-`sender` and `payload` fields are required for a valid envelope. If a valid MQTT message is found, the message is sent over the radio as a message of type `TEXT_MESSAGE_APP` with the serialized `payload` value in the message payload.
+`from` and `payload` fields are required for a valid envelope (note that in firmware <2.2.20 a field `sender` was required, but this is no longer the case). The `from` field should be equal to the node number of the node that will transmit the message. Optionally, you can specify a destination by setting the `to` field to the node number of the destination. If the `to` field is not set, the message will be broadcast to all nodes on the mesh.
```

Comment:
This line is causing the build to fail, you might try '&lt; 2.2.20'

Reply:
Ah, sorry, missed that. Changed it and checked that it builds now.